Dawid-Skene model implementation

usptact opened the pull request:
Implementing Dawid-Skene model after (Passonneau & Carpenter, 2014).

Commit:
Merge remote-tracking branch 'refs/remotes/pymc-devs/master'

Commit:
Dawid-Skene model implementation

Implementing the model from (Passonneau & Carpenter, 2014)

twiecki commented:
This looks great. The last cell does not show anything.

Also, can you add this to `docs/source/examples.rst` under `Applied`? Also, feel free to add your name and a copyright to the top.

Commit:
Minor notebook improvements and adding entry in examples.rst

usptact commented:
@twiecki I slightly improved the wording in the notebook and added an entry in the examples.rst file. I changed the last cell to print confusion matrices, not plot (strange, it worked on my Win64 machine).

twiecki commented:
Thanks @usptact!
